perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -465,6 +475,37 @@ def _check_dependencies(packages) -> None:
 
   return
 
+def _extract_vm_metrics(results_list, folders, mount_type) -> list:
+  """Extracts VM metrics for testing folders '1KB_100000files_0subdir' and testing_folder.name
+   for mount type 'gcs_bucket'
+
+  Args:
+    results_list (list): List containing the start and end times
+                        (for all samples) for each testing folder.
+    folders (list): List containing protobufs of testing folders.
+    mount_type (str): Access type of folder. It can take 2 values:
+                      'gcs_bucket' or 'persistent_disk'
+  Returns:
+    list: A list of extracted metrics
+  """
+  vm_metrics_obj = vm_metrics.VmMetrics()
+  vm_metrics_data = {}
+  # Getting VM metrics for listing tests on each folder
+  for testing_folder in folders:
+    # Getting start and end times for all 30 samples
+    start_time_first_sample = results_list[testing_folder.name][0][0]
+    end_time_last_sample = results_list[testing_folder.name][-1][-1]
+    metrics_data = []
+
+    if((testing_folder.name == '1KB_100000files_0subdir' or testing_folder.name == '1KB_200000files_0subdir') and mount_type == 'gcs_bucket'):
+      time.sleep(60)
```

Comment:
Why this sleep is required? Anyway we are waiting before calling this method.

Reply:
And if it is required, please write one liner comment for that.

Reply:
Thank you for pointing this out! This delay should be given when we're running the tests for gcs bucket, so that the results are more accurate, similar to load tests. I'll write a comment about it.